components/infobox/wikis/tft/infobox_league_custom.lua

```diff
@@ -105,6 +110,11 @@ function CustomLeague:_createPatchCell(args)
 	return content .. ' &ndash; [[Patch ' .. args.epatch .. '|'.. args.epatch .. ']]'
 end
 
+function CustomLeague:addToLpdb(lpdbData, args)
```

Comment:
have removed the addtolpdb, forgot commons already has it

> do you also want to display it?

Yes, Currently the **Game Version:** does displayed already in infobox
just only if **|game=** isnt input, the **game** is left empty unstored

![image](https://github.com/Liquipedia/Lua-Modules/assets/88981446/34849eb2-520d-4fd5-9218-6287262e0a67)

Reply:
okay more precise: do you want the default game displayed if no game is entered?
it makes a difference for storage because if you also want to display it you can just set a default in the template else you would need to actually adjust storage stuff a bit

Reply:
> okay more precise: do you want the default game displayed if no game is entered? it makes a difference for storage because if you also want to display it you can just set a default in the template (`|game={{#if:{{{game|}}}|{{{game}}}|hereTheDefaultGame}}`) else you would need to actually adjust storage stuff a bit


Yes, if no game is entered, default should always displayed (this part on infobox)

I guess the PR is now ready then, I made this change to the template call https://liquipedia.net/tft/index.php?title=Template%3AInfobox_league&type=revision&diff=71766&oldid=69141

Reply:
i see no commit removing the addtolpdbdata again